be/src/storage/rowset/rowset.cpp

```diff
@@ -144,8 +144,8 @@ std::string Rowset::segment_upt_file_path(const std::string& dir, const RowsetId
 }
 
 std::string Rowset::delta_column_group_path(const std::string& dir, const RowsetId& rowset_id, int segment_id,
-                                            int64_t version) {
-    return strings::Substitute("$0/$1_$2_$3.cols", dir, rowset_id.to_string(), segment_id, version);
+                                            int64_t version, int idx) {
+    return strings::Substitute("$0/$1_$2_$3_$4.cols", dir, rowset_id.to_string(), segment_id, version, idx);
```

Comment:
Will there is a compatible problem here ? if old `.cols` is `$0/$1_$2_$3.cols` ?